docs/administration/managing-users-and-teams/system-and-space-permissions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: System and Space Permissions
+description: An explanation of the two levels that permissions can apply at, the Space and the System.
+---
+
+Octopus Deploy supports partitioning your server up into [Spaces](../spaces) which enables teams to stay focused on only the projects and content that matter to those teams. As a result, permission scoping needs to respect boundaries that support both the administration of the whole server, as well as support the administration of an individual space.
+
+This introduces some complexity that can be useful to understand when things don't work quite the way you expect. Reading this page should give you a general understanding of how permissions work in these two contexts. 
+
+## Levels of permission
```

Comment:
I think this section is very well written

Reply:
:bowing_man: